### DISCUSSION TITLE
Add indicator for the status of drm on videos.

### DISCUSSION THREAD

**Discussion by Jurtaani** [f1multiviewer/issue-tracker#135] [Feature requests] [answered]
Some people might not be able to watch the drm enabled live streams or archives, because of widevine not allowing GPU accelerated video playback and their cpu not being strong enough to handle it on its own.
So to avoid having to open the stream and see if it stutters or not, there should be some kind of indicator that told if video is still under drm or not.

**Comment by f1multiviewer** [comment, marked as answer]
This is shown in the debug data, now. Unfortunately, there's no way to tell unless you request the playback URL, which isn't something I'm comfortable doing for all streams visible on a page :)
